deepchecks/nlp/text_data.py

```diff
@@ -733,6 +742,52 @@ def is_sampled(self, n_samples: t.Optional[int]):
             return False
         return self.n_samples > n_samples
 
+    def describe(self, n_properties_to_show: t.Optional[int] = 4, properties_to_show: t.Optional[t.List[str]] = None):
+        """Provide holistic view of the data.
+
+        Generates the following plots:
+        1. Label distribution
+        2. Statistics about the data such as number of samples, annotation ratio, list of metadata columns, list of
+        text properties and so on.
+        3. Property distribution for the text properties defined either by n_properties_to_show or properties_to_show
+        parameter.
+
+        Parameters
+        ----------
+        n_properties_to_show : int, default 4
```

Comment:
add parameter of maximum number of labels to show

Reply:
Done